index.html

```diff
@@ -343,7 +369,7 @@
 			var view = new MapView({
 				container: "viewDiv",
 				map: map,
-				center: [106.8033387, 10.8739831],
+				center: [106.68216950311093,10.772966281258045],

```

Comment:
This line is not in your scope. Please rollback it. Thanks.

Reply:
Done